### PR TITLE
add chinaeast2 and chinanorth2

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**: Adds chinaeast2 and chinanorth2 mooncake regions

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
add chinaeast2 and chinanorth2
```

### CODE REVIEW

**Comment by jackfrancis** [comment]
@karataliu @andyzhangx @feiskyer in case you have any Mooncake expertise to move this along, thanks!

File: pkg/acsengine/Get-AzureConstants.py [hunk 1]
```diff
@@ -86,6 +88,8 @@ def getFileContents(dcosMasterMap, masterAgentMap, kubernetesAgentMap, sizeMap,
 
     text += r"""        "chinaeast",
 	"chinanorth",
+    "chinanorth2",
+    "chinaeast2",
```
Comment (feiskyer) [inline pkg/acsengine/Get-AzureConstants.py]: nit: indent with others

**Comment by jackfrancis** [comment]
Thanks for the review @feiskyer and @andyzhangx!

**Pull request merged**